src/classes/services/HttpService.cls

```diff
@@ -0,0 +1,22 @@
+/**
+ * Created by Mariya on 30.11.2022.
+ */
+
+public with sharing class HttpService {
```

Comment:
MINOR
This service class does not need `with sharing` keyword as there is no data operations.